CAIPs/caip-25.md

```diff
@@ -127,24 +157,25 @@ The wallet can respond to this method with either a success result or an error m
 
 #### Success
 
-The succesfull reslt contains one mandatory string (keyed as `sessionId` with a value 
+The successful result contains one mandatory string (keyed as `sessionId` with a value 
 conformant to [CAIP-171][]) and two session objects, both mandatory and non-empty. 
 
-The first is called `sessionNamespaces` and contains 1 or more namespace objects.
-* All required namespaces and all, none, or some of the optional namespaces (at the 
-discretion of the provider) MUST be included if successful.  
-* As in the request, each namespace object MUST contain `methods` and `events` objects, 
-and a `chains` object if a specific chain is not specified in the object's index.
-* Unlike the request, each namespace object MUST also contain an `accounts` array, 
-containing 0 or more [CAIP-10][] conformant accounts authorized for the session and valid
-in the namespace and chain(s) authorized by the object they are in. Additional constraints
-on the accounts authorized for a given session MAY be specified in the corresponding 
-[namespaces][] specification.
-
-A `sessionProperties` object MAY also be present, and its contents MAY correspond to the
-properties requested in the response or not (at the discretion of the provider) but MUST
-conform to the properties names and value constraints described in [CAIP-170][]; any other 
-MUST be dropped by the requester.
+The first is called `sessionScopes` and contains 1 or more scope objects.
+* All required scope objects and all, none, or some of the optional scope object
+(at the discretion of the provider) MUST be included if successful.  
+* As in the request, each scope object object MUST contain `methods` and
+`events` objects, and a `chains` object if a specific chain is not specified in
```

Comment:
```suggestion
`notifications` objects, and a `chains` object if a specific chain is not specified in
```
Should we just change this to `notifications` to mirror CAIP-211? I think we all settled on `notifications` over `events`.

Reply:
Actually all the "events" become "notifications" in 211 I think!
https://github.com/ChainAgnostic/CAIPs/pull/211/files

Reply:
It might lead to some merge conflicts with 211 but yeah, happy to rename them all here.  In all the new text in 211, they're notifications, not events :D

Reply:
see most recent commit